### ISSUE TITLE
Write a series of test reports

### ISSUE THREAD

**Issue by ftrotter** [CareSet/Zermelo#41]
Write a series of test reports. 

The test reports should rely on the same test data as [DURC test data](https://github.com/CareSet/DURC/tree/master/test_databases) and/or the mysql meta information tables which should always be there. 

The tests should include their purpose and what the expected result should be in the test report. They should basically test any reasonable parameter tweak that a given report can achieve. 

For instance, OverrideHeader can dramatically change how a given column header is displayed and how it works. There should be one report that just exercises these permutations. Demonstrating how they work and also allowing us as developers to quickly see that they still work as expected. 

 MapRow should enable the creation of rows with bootstrap buttons or other html form components, as well as functioning links to other sites. There should be one report that just exercises this very heavily. 

Basically this task is to start to use "reports as unit tests" for the project.

**Comment by ftrotter** [comment]
This has been done
